### PR TITLE
Remove pagerfanta dependency

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no 
| New feature? | no 
| BC breaks? | no 
| Deprecations? | no  
| Fixed tickets | fixes # 
| Related issues/PRs | # 
| License | MIT
| Documentation PR | sulu/sulu-docs# 

#### What's in this PR?

Remove pagerfanta dependency.

#### Why?

Lesser dependency will make it easier to maintain sulu.

### CODE REVIEW

**Comment by niklasnatter** [comment]
Tests are failing 😕

**Comment by niklasnatter** [comment]
Still failing 😢

File: src/Sulu/Bundle/SearchBundle/Controller/SearchController.php [hunk 1]
```diff
@@ -110,21 +108,24 @@ function(string $indexName) use ($indexNames) {
...
+        $result = $query->execute();
+        $total = $result->getTotal();
 
-        $adapter = new ArrayAdapter(\iterator_to_array($query->execute()));
-        $pager = new Pagerfanta($adapter);
-        $pager->setMaxPerPage($limit);
-        $pager->setCurrentPage($page);
+        // FIXME
+        //    zend_lucene seems not to have any pagination implemented and will return here
+        //    the total query hits we paginate in php here to return the correct results
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/SearchBundle/Controller/SearchController.php:119]: `pagerfanta` was here used because of this issue

**Comment by alexander-schranz** [comment]
@nnatter ready for review :)

File: src/Sulu/Bundle/SearchBundle/Tests/Functional/Controller/SearchControllerTest.php [hunk 2]
```diff
@@ -155,6 +155,46 @@ public function provideSearch()
                     'total' => 2,
                 ],
             ],
+            [
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/SearchBundle/Tests/Functional/Controller/SearchControllerTest.php:158]: added this test case that page 1 and page 2 is tested to get the correct result.

**Pull request merged**